### ISSUE TITLE
Missing contracts?

### ISSUE THREAD

**Issue by Benjythebee** [itsjerryokolo/OpenSea#1]
Heya, this is pretty cool.
I tried running the following query on the explorer:
```
{
  contract(id:"0x79986af15539de2db9a5086382daeda917a9cf0c") {
    id
    address
    name
  }
}

```
with `0x79986af15539de2db9a5086382daeda917a9cf0c` being the Cryptovoxels contract address.
the query returns `null` which doesn't seem right;

**Comment by ACollectionOfAtoms** [comment]
Yeah, not sure this subgraph is fully function. The `NFT` object doesn't return expected values for id either.

**Comment by itsjerryokolo** [comment]
@Benjythebee @ACollectionOfAtoms This subgraph isn't complete. Currently, `Collections` or `Contracts` is missing. I can't seem to find the contract that handles that.

**Comment by yujianFresh** [comment]
> @Benjythebee @ACollectionOfAtoms This subgraph isn't complete. Currently, `Collections` or `Contracts` is missing. I can't seem to find the contract that handles that.

hi bro, you know fully function subgraph of opensea ? if have, tell me , thanks
